src/main/js/dynamic/DistributionConfiguration.js

```diff
@@ -141,6 +145,18 @@ class DistributionConfiguration extends Component {
         this.props.testDistributionJob(jsonBody);
     }
 
+    handleSendMessage() {
+        this.setState({
+            showSendMessage: true
+        });
+    }
+
+    hideSendMessage() {
+        this.setState({
+            showSendMessage: false
+        });
+    }
```

Comment:
We can probably pass in the boolean as a parameter. When calling the fuction as a prop, we should be able to pass true or false.

Reply:
Not a bad idea.